Cargo.toml

```diff
@@ -145,7 +145,9 @@ rev = "7dd3d42f0c21384950afe6a46ed85352acf71625"
 [dev-dependencies]
 # See https://bheisler.github.io/criterion.rs/book/user_guide/known_limitations.html for the usage
 # of `real_blackbox` feature.
-criterion = { version = "0.2.11", features=['real_blackbox'] }
+criterion = "0.3"
+criterion-papi = "0.1"
```

Comment:
I see you made this! So cool! Congratulations! (https://github.com/YangKeao/criterion-papi)